.travis.yml

```diff
@@ -1,14 +1,33 @@
-language: java
-
-jdk:
-- oraclejdk8
-
-before_install: 
-  - curl -O -L https://github.com/inventit/mqtt-websocket-java/releases/download/1.0.1/mqtt-websocket-java-1.0.1.jar
-  - mvn install:install-file -Dfile=mqtt-websocket-java-1.0.1.jar -DgroupId=io.inventit.dev -DartifactId=mqtt-websocket-java -Dversion=1.0.1 -Dpackaging=jar
-
-install: "mvn -Dmaven.test.skip=true clean install --batch-mode"
-script: "mvn -Djava.awt.headless=true -Dmaven.test.redirectTestOutputToFile=true -Dcobertura.report.format=xml --fail-at-end --batch-mode cobertura:cobertura test "
-
-after_success:
+env:
+  global:
+    - MAVEN_OPTS=-Dmaven.repo.local=.m2/repository
+    - MQTT_WEBSOCKET_VERSION=1.0.1
+    
+cache:
+  directories:
+  - .jmeter
+  - .m2
+
+language: java
+
+jdk:
+- oraclejdk8
+
+before_install: 
+  - sudo apt-get -y update
+  - sudo apt-get -y install python default-jre-headless python-tk python-pip python-dev libxml2-dev libxslt-dev zlib1g-dev net-tools
+  - sudo pip install bzt==1.11.0
+  - chmod -R +x ./.travis
+  - "sh ./.travis/install-maven-dependency.sh https://github.com/inventit/mqtt-websocket-java/releases/download/$MQTT_WEBSOCKET_VERSION/mqtt-websocket-java-$MQTT_WEBSOCKET_VERSION.jar io.inventit.dev mqtt-websocket-java $MQTT_WEBSOCKET_VERSION"
+
+
+script:
+  - "mvn -DskipTests=true clean install --batch-mode"
+  - "mvn -Djava.awt.headless=true -Dmaven.test.redirectTestOutputToFile=true -Dcobertura.report.format=xml --fail-at-end --batch-mode cobertura:cobertura test"
+  - "sh ./http2/target/jmeter-test/testJMeter.sh 4.0 $HOME/.jmeter"
+  - "sh ./http2/target/jmeter-test/testJMeter.sh 3.3 $HOME/.jmeter"
+  - "sh ./http2/target/jmeter-test/testJMeter.sh 3.2 $HOME/.jmeter"
+  - "sh ./http2/target/jmeter-test/testJMeter.sh 3.1 $HOME/.jmeter"

```

Comment:
What about testing 3.0 and 2.13?

Reply:
These were the jmeter versions that were requested to support for the plugin. Do you think might it be worth it to add tests for 3.0 and 2.13 as well? Adding more tests would imply longer build times, and we might find some nasty issues with 2.13 (since is so old and major version change) that may not be worth to fix.

Reply:
Now me and @undera  support all versions since 2.13

If you have not requests for support the oldest versions, possible we can skip it. 

The longer build time does not matter for Travis.

Reply:
done